app/views/spree/admin/orders/bulk_management.html.haml

```diff
@@ -50,38 +56,55 @@
     .row
       .one.column.alpha &nbsp;
       .two.columns
-        %span.two.columns Group Buy Unit Size
+        %span.two.columns 
+          = t "group_buy_unit_size"
         %span.two.columns {{ formattedValueWithUnitName( selectedUnitsProduct.group_buy_unit_size, selectedUnitsProduct, selectedUnitsVariant ) }}
       .one.column &nbsp;
       .two.columns
-        %span.two.columns Total Quantity Ordered
+        %span.two.columns
+          = t "total_qtt_ordered"
         %span.two.columns {{ formattedValueWithUnitName( sumUnitValues(), selectedUnitsProduct, selectedUnitsVariant ) }}
       .one.column &nbsp;
       .two.columns
-        %span.two.columns Max Quantity Ordered
+        %span.two.columns
+          = t "max_qtt_ordered"
         %span.two.columns {{ formattedValueWithUnitName( sumMaxUnitValues(), selectedUnitsProduct, selectedUnitsVariant ) }}
       .one.column &nbsp;
       .two.columns
-        %span.two.columns Current Fulfilled Units
+        %span.two.columns
+          = t "current_fulfilled_units"
         %span.two.columns {{ fulfilled(sumUnitValues()) }}
       .one.column &nbsp;
       .two.columns
-        %span.two.columns Max Fulfilled Units
+        %span.two.columns
+          = t "max_fulfilled_units"
         %span.two.columns {{ fulfilled(sumMaxUnitValues()) }}
       .one.column.omega &nbsp;
     %div{ :class => "eight columns alpha", 'ng-hide' => 'allFinalWeightVolumesPresent()' }
       %span{ :class => "eight columns alpha", style: 'color:red' }
-        WARNING: Some variants do not have a unit value
-
-  %hr.divider.sixteen.columns.alpha.omega
-
-  .controls.sixteen.columns.alpha.omega{ ng: { hide: 'RequestMonitor.loading || lineItems.length == 0' } }
-    %div.three.columns.alpha
-      %input.fullwidth{ :type => "text", :id => 'quick_search', 'ng-model' => 'quickSearch', :placeholder => 'Quick Search' }
-    = render 'admin/shared/bulk_actions_dropdown'
-    %div.seven.columns &nbsp;
-    = render 'admin/shared/columns_dropdown'
-
+        = t "bulk_management_warning"
```

Comment:
This bit is a bit more than translation. Have the replaced files been deleted? E.g. admin/shared/_bulk_actions_dropdown.html.haml`

Was this place the only one using these shared files?